test/helpers/framework.go

```diff
@@ -60,6 +60,8 @@ type LogStore interface {
 	HasInfraStructureLogs(timeToWait time.Duration) (bool, error)
 
 	HasAuditLogs(timeToWait time.Duration) (bool, error)
+
+	ClusterLocalEndpoint() string
```

Comment:
What does this represent? The name is not obvious to me

Reply:
It means you provide a service object for your store and this method gives you the cluster local DNS name e.g. kafka.openshift-logging.svc.cluster.local